packages/lib/server-only/document/send-document.tsx

```diff
@@ -213,31 +212,6 @@ export const sendDocument = async ({
     }),
   );
 
-  const allRecipientsHaveNoActionToTake = document.Recipient.every(
```

Comment:
@adithyaakrishna not sure I get it, won't removing this not break the "every signer is a BCC" case?

Reply:
Closing this PR and opening another one, this has some changes not done by me :/